packages/logger/tests/e2e/basicFeatures.middy.test.FunctionCode.ts

```diff
@@ -34,6 +36,14 @@ const testFunction = async (event: APIGatewayProxyEvent, context: Context): Prom
     logger.error(ERROR_MSG, e as Error);
   }
 
+  // Test feature 7: Logging an arbitrary object
+  const obj: APIGatewayProxyResult = {
+    body: ARBITRARY_OBJECT_DATA,
```

Comment:
Improvement: by looking at this PR, the type of the body value will be a string or undefined. 
One of the goals of this PR would be allow for logging of nested objects. I suggest we use that scenario instead.